app/presenters/reservation_calendar.rb

```diff
@@ -0,0 +1,19 @@
+class ReservationCalendar < SimpleDelegator
+
+  def initialize(reservation)
+    super(reservation)
+  end
+
+  def as_ical
+    cal = Icalendar::Calendar.new
+    cal.event do |e|
+      e.dtstart = Icalendar::Values::DateTime.new(reserve_start_at)
```

Comment:
I'm not quite sure what you mean by time zone conversion. I would guess that as long as the timezone gets rendered out as part of the dtstart/dtend, then it should be fine. I'm not quite sure how icalendar handles timezones.